From ce23169a09cc0084512ec44c5f08cb8614c9e653 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20B=C3=BCrkler?= <andreas.buerkler@enclustra.com>
Date: Wed, 29 Jan 2025 12:49:09 +0000
Subject: [PATCH] Use only high memory region

---
 arch/riscv/cpu/mpfs/dram.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/riscv/cpu/mpfs/dram.c b/arch/riscv/cpu/mpfs/dram.c
index 509af55804..2fbda7d990 100644
--- a/arch/riscv/cpu/mpfs/dram.c
+++ b/arch/riscv/cpu/mpfs/dram.c
@@ -31,6 +31,9 @@ phys_size_t board_get_usable_ram_top(phys_size_t total_size)
 	 * U-Boot is cached addresses, but also account for the reservation at
 	 * the top of 32 bit cached DDR used by the HSS.
 	 */
+#ifdef CONFIG_TARGET_ENCLUSTRA_MERCURY_MP1
+	return gd->ram_top;
+#endif
 	if (gd->ram_top >= MPFS_TOP_OF_CACHED - MPFS_HSS_RESERVATION)
 		return MPFS_TOP_OF_CACHED - MPFS_HSS_RESERVATION - 1;
 	/*
-- 
2.34.1

