From d1f8b1408293b86dde8180f6935a8598620fd1dd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20B=C3=BCrkler?= <andreas.buerkler@enclustra.com>
Date: Mon, 27 Jan 2025 19:06:13 +0000
Subject: [PATCH] PolarFire SoC I2C driver modification for zero sized

---
 drivers/i2c/i2c-microchip.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/i2c/i2c-microchip.c b/drivers/i2c/i2c-microchip.c
index d82b80f535..97faf208ea 100644
--- a/drivers/i2c/i2c-microchip.c
+++ b/drivers/i2c/i2c-microchip.c
@@ -376,9 +376,6 @@ static int mpfs_i2c_xfer_msg(struct mpfs_i2c_bus *bus, struct i2c_msg *msg)
 	u8 ctrl;
 	int ret;
 
-	if (!msg->len || !msg->buf)
-		return -EINVAL;
-
 	bus->addr = i2c_8bit_addr_from_msg(msg);
 	bus->msg_len = msg->len;
 	bus->buf = msg->buf;
-- 
2.34.1

