From a72321e7c25346c87d4e7ce979e63c7df01f5b02 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20B=C3=BCrkler?= <andreas.buerkler@enclustra.com>
Date: Mon, 27 Jan 2025 19:18:41 +0000
Subject: [PATCH] Use only high memory region

---
 arch/riscv/cpu/generic/dram.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/riscv/cpu/generic/dram.c b/arch/riscv/cpu/generic/dram.c
index 44e11bd56c..e93a3d429a 100644
--- a/arch/riscv/cpu/generic/dram.c
+++ b/arch/riscv/cpu/generic/dram.c
@@ -23,6 +23,7 @@ int dram_init_banksize(void)
 
 phys_size_t board_get_usable_ram_top(phys_size_t total_size)
 {
+#if !defined(CONFIG_TARGET_ENCLUSTRA_MERCURY_MP1)
 	/*
 	 * Ensure that we run from first 4GB so that all
 	 * addresses used by U-Boot are 32bit addresses.
@@ -33,6 +34,7 @@ phys_size_t board_get_usable_ram_top(phys_size_t total_size)
 	 */
 	if (gd->ram_top >= SZ_4G)
 		return SZ_4G - 1;
+#endif
 
 	return gd->ram_top;
 }
-- 
2.34.1

